root/io/io/MainEventCompressionBenchmark.cxx

```diff
@@ -0,0 +1,119 @@
+#include "TFile.h"
+#include "TTree.h"
+
+#include "benchmark/benchmark.h"
+#include "rootbench/RBConfig.h"
+
+static std::string GetAlgoName(int algo) {
+   std::string algo_name = "";
+   if (algo == 0) {
+      algo_name = "zlib";
+   } else if (algo == 2) {
+      algo_name = "lzma";
+   } 
+   else if (algo == 4) {
+      algo_name = "lz4";
+   } 
+   else if (algo == 5) {
+      algo_name = "zstd";
+   }
+   return algo_name;
+}
+
+static void BM_MainEvent_Compress(benchmark::State &state, int algo) {
```

Comment:
We will need to rename benchmarks here, please check https://github.com/root-project/rootbench#simple-benchmark-template

Reply:
Could you please provide an example of what name should it have?

Reply:
Maybe BM_TFile_MainEventCompress? (should be BM_CLASSNAME_ROUTINE)